css/icons.css

```diff
@@ -0,0 +1,52 @@
+.se-icon {
+    font-size: 24px;
+    line-height: 1 !important;
+    display: inline-block !important;
+    vertical-align: middle !important;
+    background: none !important;
+    width: 1em !important;
+    height: 1em !important;
+    text-decoration: none !important;
+}
+
+.se-icon::before {
+    display: inline-block;
+    width: 1em;
+    height: 1em;
+}
+
+.se-hidden-icon {
+    display: none !important;
+}
+
+.se-remove-icon {
+    background-image: none !important;
+    width: unset !important;
+    height: unset !important;
+    text-indent: unset !important;
+}
+
+.se-remove-icon::before {
+    display: none;
+    background-image: none !important;
+    content: none !important;
+    width: unset !important;
+    height: unset !important;
+}
+
+#infopessoalh:not(.expandido) .colapsar {
+    display: none
+}
+
+#infopessoalh.expandido .expandir {
+    display: none
+}
+
+div.info,
+div.alerta {
+    background-image: none;
+    display: flex;
+    align-items: center;
+    gap: .5em;
+    padding-left: .5em;
+}
```

Comment:
Can you remove this?
This will be covered by #30

Reply:
Done!